beam-migrate/Database/Beam/Migrate/Serialization.hs

```diff
@@ -322,7 +325,11 @@ sql92Deserializers = mconcat
                    , beamDeserializer deserializeSql92Attributes ]
   where
     parseSub nm o key parse =
+#if MIN_VERSION_aeson(2,0,0)
+      withObject (unpack (nm <> "." <> key)) parse =<< o .: DAK.fromText key
```

Comment:
nit: I prefer to avoid duplication in CPP selections as much as possible. So something like this:

```
#if MIN_VERSION_aeson(2,0,0)
    makeKey = DAK.fromText
#else
    makeKey = id
#endif
    parseSub nm o key parse =
      withObject (unpack (nm <> "." <> key)) parse =<< o .: makeKey key
```

Reply:
:+1: